mlflow/store/model_registry/file_store.py

```diff
@@ -78,6 +81,31 @@ def _validate_model_name(name):
         )
 
 
+class FileModelVersion(ModelVersion):
+    def __init__(self, storage_location=None, **kwargs):
+        super().__init__(**kwargs)
+        self._storage_location = storage_location
+
+    @property
+    def storage_location(self):
+        """String. The storage location of the model version."""
+        return self._storage_location
+
+    @storage_location.setter
+    def storage_location(self, location):
+        self._storage_location = location
+
+    @classmethod
+    def _properties(cls):
+        # aggregate with parent class with subclass properties
+        return sorted(ModelVersion._properties() + cls._get_properties_helper())
```

Comment:
Does `_get_properties_helper` also return the properties define in `ModelVersion`?

Reply:
Nope. I was hoping it would but it only returned the properties in `FileModelVersion`, i.e. only `storage_location`.